app/controllers/provider/admin/account/email_configurations_controller.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+class Provider::Admin::Account::EmailConfigurationsController < Provider::Admin::Account::BaseController
+
+  # We only enable it for master for now, feel free to roll it out for any provider
+  before_action :ensure_master_domain, :email_configurations_enabled?
```

Comment:
This is already defined in a parent class Provider::Admin::BaseController: https://github.com/3scale/porta/blob/25fa9bc53bdd3eeb8f2a9c8e882f771c85c3c3cb/app/controllers/provider/admin/base_controller.rb#L2